ozy/__main__.py

```diff
@@ -77,7 +77,7 @@ def show_path_warning(ozy_bin_dir):
     _LOGGER.info("  bash$ echo -e '# ozy support\\nexport PATH=%s:$PATH' >> ~/.bashrc", ozy_bin_dir)
     _LOGGER.info("  then restart your shell sessions")
     _LOGGER.info("zsh shell users:")
-    _LOGGER.info("  zsh$ # some kind of wizardry here TODO!")
+    _LOGGER.info(f"  zsh$ # path+=({ozy_bin_dir})\\nexport PATH")
```

Comment:
Fantastic: thanks. Does this persist forever once you've done it? I don't know much about `zsh`

Reply:
it doesn't - you still have to update your  zshrc